pyvista/core/filters/data_set.py

```diff
@@ -1578,40 +1575,40 @@ def contour(
 
         Parameters
         ----------
-        isosurfaces : int or sequence, optional
+        isosurfaces : int | sequence[float], optional
             Number of isosurfaces to compute across valid data range or a
             sequence of float values to explicitly use as the isosurfaces.
 
-        scalars : str, collections.abc.Sequence, numpy.ndarray, optional
+        scalars : sequence[float], optional
```

Comment:
This can also be the name, right? Perhaps
```suggestion
        scalars : str | sequence[float], optional
```
Or
```suggestion
        scalars : str | array_like[float], optional
```